examples/text_labels_and_annotations/font_family_rc_sgskip.py

```diff
@@ -4,12 +4,13 @@
 ===========================
 
 You can explicitly set which font family is picked up, either by user-defined
-family names, or generic-families (e.g., 'serif', 'sans-serif', 'monospace', etc.),
-or a combination of both (see :doc:`font tutorial </tutorials/text/text_props>`).
+family names, or generic-families (e.g., 'serif', 'sans-serif', 'monospace',
```

Comment:
there are 5 generic families only, you may as well list them all...

Reply:
👍🏼